packages/graphql/index.js

```diff
@@ -9,6 +9,8 @@ import {
   GraphQLID,
 } from 'graphql';
 
+import { get } from './src/services/HttpRequest';
```

Comment:
Promise yourself to never ever copy anything from our internal GraphQL. One week old project and you already copy-pasted our technical debt... 🙈🙂

Also, do you need anything from the repo? Let me know please and I can help you with that. Copy-paste doesn't sound like a way to go.

Reply:
In that case, please export these data fetching functions, so we can re use them here 🙂